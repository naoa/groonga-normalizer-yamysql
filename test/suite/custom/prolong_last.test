register normalizers/yamysql
normalize NormalizerYaMySQL "デ-ター" WITH_TYPES|WITH_CHECKS

