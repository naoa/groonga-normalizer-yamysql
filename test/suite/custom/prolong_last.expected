register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQL "デ-ター" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "デ-タ-",
    "types": [
      "katakana",
      "symbol",
      "katakana",
      "symbol"
    ],
    "checks": [
      3,
      0,
      0,
      1,
      3,
      0,
      0,
      3
    ]
  }
]
